src/resources/views/crud/columns/hidden.blade.php

```diff
@@ -0,0 +1,28 @@
+{{-- regular object attribute --}}
+@php
+    $column['value'] = $column['value'] ?? data_get($entry, $column['name']);
+    $column['escaped'] = $column['escaped'] ?? true;
+    $column['text'] = $column['default'] ?? '-';
+
+    if($column['value'] instanceof \Closure) {
+        $column['value'] = $column['value']($entry);
+    }
+
+    if(is_array($column['value'])) {
+        $column['value'] = json_encode($column['value']);
+    }
+
+    if(!empty($column['value'])) {
+        $column['text'] = $column['value'];
+    }
+@endphp
+
+<span>
+    @includeWhen(!empty($column['wrapper']), 'crud::columns.inc.wrapper_start')
+        @if($column['escaped'])
+            {{ $column['text'] }}
+        @else
+            {!! $column['text'] !!}
+        @endif
+    @includeWhen(!empty($column['wrapper']), 'crud::columns.inc.wrapper_end')
+</span>
```

Comment:
Munjal, I see you've added individual column types for each field (which is great), but a lot of them are the same or very similar. Basically the `text` column a lot of times. Instead of having all this duplicated code, you know what we can do? We can create "alias" columns, that will just load a different column. For example, here I think we can just do:

```suggestion
@include('crud::columns.text')
```

I think this would heavily reduce the amount of code we have to add (and maintain) and while providing the exact same functionality. Don't you think? Am I missing something that makes that not a good idea?

If you also think that would be possible and cleaner, let's please go through all fields added by this PR, and where it makes sense, instead of having new code, just include other fields we already have 💪